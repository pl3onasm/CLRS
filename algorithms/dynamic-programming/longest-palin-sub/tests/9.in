12345610456007489645650840860112604898312368740898415610308465140106541560469401530165013608438046153504621463574865106540865