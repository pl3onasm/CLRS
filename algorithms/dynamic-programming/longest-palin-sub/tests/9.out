LPS length: 73
LPS: 5604560568461264316408415610304651401041564030165148046134621648650654065